geniza/locale/ar/LC_MESSAGES/django.po

```diff
@@ -1,35 +1,33 @@
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR The Trustees of Princeton University
-# This file is distributed under the same license as the geniza package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
```

Comment:
crowdin removes some of this metadata, so I deleted the parts of our custom `makemessages` that add it, since it'll get deleted everytime we regenerate messages.